lib/generators/the_policy_machine/update_accessible_objects_filter_performance_generator.rb

```diff
@@ -0,0 +1,12 @@
+module ThePolicyMachine
+  module Generators
+    class UpdateAccessibleObjectsFilterPerformanceGenerator < Rails::Generators::Base
+      source_root File.expand_path('../../../migrations', __FILE__)
+
+      def generate_update_accessible_objects_filter_performance_migration
```

Comment:
Just out of curiosity, why do we do all this migration generation manually instead of the standard `rails gem_name:install:migrations`?

Reply:
Simple as this is the pattern the gem has been using since many years ago.